opentelemetry-api/opentelemetry/trace/__init__.py

```diff
@@ -115,40 +124,46 @@ def start_span(self, name: str, parent: 'Span') -> Iterator['Span']:
 
         is equivalent to::
 
-            span = tracer.create_span(name, parent=tracer.get_current_span())
+            span = tracer.create_span(name)
             with tracer.use_span(span):
                 do_work()
 
         Args:
             name: The name of the span to be created.
-            parent: The span's parent.
+            parent: The span's parent. Defaults to the current span.
 
         Yields:
             The newly-created span.
         """
 
-    def create_span(self, name: str, parent: 'Span') -> 'Span':
-        """Creates a new child span of the given parent.
+    def create_span(self,
+                    name: str,
+                    parent: 'Span' = CURRENT_SPAN) -> 'Span':
+        """Creates a span.
 
         Creating the span does not start it, and should not affect the tracer's
         context. To start the span and update the tracer's context to make it
         the currently active span, see :meth:`use_span`.
 
-        Applications that need to explicitly set spans' parents or create spans
-        detached from the tracer's context should use this method.
+        By default the current span will be used as parent, but an explicit
+        parent can also be specified. If the specified value is `None`, the
+        created span will be a root span.
+
+        Applications that need to create spans detached from the tracer's
+        context should use this method.
 
             with tracer.start_span(name) as span:
```

Comment:
I think this sample code is a copy & paste error. It should probably not be here (although it made more sense when the paragraph talking about `use_span` was directly before it).

Reply:
Was wondering the same.